src/Value/CSSFunction.php

```diff
@@ -17,19 +17,18 @@
 class CSSFunction extends ValueList
 {
     /**
-     * @var string
+     * @var non-empty-string
      *
      * @internal since 8.8.0
      */
     protected $name;
 
     /**
-     * @param string $name
-     * @param RuleValueList|array<array-key, Value|string> $arguments
-     * @param string $separator
```

Comment:
Can this be a `non-empty-string` as well?

Reply:
It can indeed, and should be. Changed.